From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Tom Levy <tomlevy93@gmail.com>
Date: Sat, 13 Jun 2020 23:41:59 +1200
Subject: backport: Change team.icpcid to externalid in NZPCDataFixtureFixes

The property was renamed from icpc to externalid in commit 0356a0469.
---
 webapp/src/DataFixtures/NZPCFixtureFixes.php | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/webapp/src/DataFixtures/NZPCFixtureFixes.php b/webapp/src/DataFixtures/NZPCFixtureFixes.php
index 8baf49f04..d681a1575 100644
--- a/webapp/src/DataFixtures/NZPCFixtureFixes.php
+++ b/webapp/src/DataFixtures/NZPCFixtureFixes.php
@@ -49,7 +49,7 @@ class NZPCFixtureFixes extends Fixture
         // categories created by the fixtures.
 
         // update example team
-        $team = $manager->getRepository(Team::class)->findOneBy(['icpcid' => 'exteam']);
+        $team = $manager->getRepository(Team::class)->findOneBy(['externalid' => 'exteam']);
         if (!$team) {
             $this->logger->warning('Cannot find example team "exteam"');
         } else {
-- 
2.20.1

